gst-plugin-pravega/src/pravegasink/imp.rs

```diff
@@ -281,6 +424,41 @@ impl ObjectImpl for PravegaSink {
                 None,
                 glib::ParamFlags::WRITABLE,
             ),
+            glib::ParamSpec::new_enum(
+                PROPERTY_NAME_RETENTION_TYPE,
+                "Retention type",
+                "If 'days', data older than 'retention-days' will be deleted from the stream. If 'bytes', the oldest data will be deleted so that the data size does not exceed 'retention-bytes'. If daysAndBytes, the oldest data will be deleted if it is older than retention-days or the data size exceeds retention-bytes.",
+                RetentionType::static_type(),
+                DEFAULT_RETENTION_TYPE as i32,
+                glib::ParamFlags::WRITABLE,
+            ),
+            glib::ParamSpec::new_double(
+                PROPERTY_NAME_RETENTION_DAYS,
+                "Retention days",
+                "The number of days that the video stream will be retained.",
+                0.0,
+                std::f64::INFINITY,
+                0.0,
+                glib::ParamFlags::WRITABLE,
+            ),
+            glib::ParamSpec::new_uint64(
+                PROPERTY_NAME_RETENTION_BYTES,
+                "Retention bytes",
+                "The number of bytes that the video stream will be retained.",
+                0,
+                std::u64::MAX,
+                0,
+                glib::ParamFlags::WRITABLE,
+            ),
+            glib::ParamSpec::new_uint64(
+                PROPERTY_NAME_RETENTION_MAINTENANCE_INTERVAL_SECONDS,
+                "Retention maintenance interval seconds",
+                "The period of the retention maintenance job.",
```

Comment:
Change to "The oldest data will be deleted from the stream with this interval, according to the retention policy."

Reply:
changed.